File: 001_two_sum/two_sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
    let ans = []
    nums.forEach((value, index) => {
        nums.slice(index + 1).forEach((value1, index1) => {
            if (value + value1 == target) {
                ans.push(index, nums.indexOf(value1, index + 1))
            }
        })
    })
    return ans
};

File: 001_two_sum/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        ans = []
        for idx, value in enumerate(nums):
            for idx1, value1 in enumerate(nums[idx + 1 :]):
                if value + value1 == target:
                    ans.extend([idx, nums.index(value1, idx + 1)])
        return ans
